packages/inter-protocol/scripts/add-collateral-core.js

```diff
@@ -10,7 +10,11 @@ import { makeInstallCache } from '../src/proposals/utils.js';
 
 export const defaultProposalBuilder = async (
   { publishRef, install: install0, wrapInstall },
-  { interchainAssetOptions = /** @type {object} */ ({}) } = {},
+  {
+    debtLimitValue = undefined,
+    interestRateValue = undefined,
```

Comment:
here's hoping for a uniform way to treat such config params:

 - #7309

Reply:
someday!